worlds/hades/Locations.py

```diff
@@ -0,0 +1,442 @@
+    "UrnsOfWealth1Location": max_number_room_checks+36,
+    "UrnsOfWealth2Location": max_number_room_checks+37,
+    "UrnsOfWealth3Location": max_number_room_checks+38,
+    "InfernalTrove1Location": max_number_room_checks+39,
+    "InfernalTrove2Location": max_number_room_checks+40,
+    "InfernalTrove3Location": max_number_room_checks+41,
+    "KeepsakeCollectionLocation": max_number_room_checks+42,
+}
+
+location_store_diamonds ={
+    "DeluxeContractorDeskLocation": max_number_room_checks+44,
+    "VanquishersKeepLocation": max_number_room_checks+45,
+    "FishingRodLocation": max_number_room_checks+46,
+    "CourtMusicianSentenceLocation": max_number_room_checks+47,
+    "CourtMusicianStandLocation": max_number_room_checks+48,
+    "PitchBlackDarknessLocation": max_number_room_checks+49,
+    "FatedKeysLocation": max_number_room_checks+50,
+    "BrilliantGemstonesLocation": max_number_room_checks+51,
+    "VintageNectarLocation": max_number_room_checks+52,
+    "DarkerThirstLocation": max_number_room_checks+53, 
+}
+
+location_table_fates = {
+    "IsThereNoEscape?": max_number_room_checks+54,
+    "DistantRelatives": max_number_room_checks+55,
+    "ChthonicColleagues": max_number_room_checks+56,
+    "TheReluctantMusician": max_number_room_checks+57,
+    "GoddessOfWisdom": max_number_room_checks+58,
+    "GodOfTheHeavens": max_number_room_checks+59,
+    "GodOfTheSea": max_number_room_checks+60,
+    "GoddessOfLove": max_number_room_checks+61,
+    "GodOfWar": max_number_room_checks+62,
+    "GoddessOfTheHunt": max_number_room_checks+63,
+    "GodOfWine": max_number_room_checks+64,
+    "GodOfSwiftness": max_number_room_checks+65,
+    "GoddessOfSeasons": max_number_room_checks+66,
+    "PowerWithoutEqual": max_number_room_checks+67,
+    "DivinePairings": max_number_room_checks+68,
+    "PrimordialBoons": max_number_room_checks+69,
+    "PrimordialBanes": max_number_room_checks+70,
+    "InfernalArms": max_number_room_checks+71,
+    "TheStygianBlade": max_number_room_checks+72,
+    "TheHeartSeekingBow": max_number_room_checks+73,
+    "TheShieldOfChaos": max_number_room_checks+74,
+    "TheEternalSpear": max_number_room_checks+75,
+    "TheTwinFists": max_number_room_checks+76,
+    "TheAdamantRail": max_number_room_checks+77,
+    "MasterOfArms": max_number_room_checks+78,
+    "AViolentPast": max_number_room_checks+79,
+    "HarshConditions": max_number_room_checks+80,
+    "SlashedBenefits": max_number_room_checks+81,
+    "WantonRansacking": max_number_room_checks+82,
+    "ASimpleJob": max_number_room_checks+83,
+    "ChthonicKnowledge": max_number_room_checks+84,
+    "CustomerLoyalty": max_number_room_checks+85,
+    "DarkReflections": max_number_room_checks+86,
+    "CloseAtHeart": max_number_room_checks+87,
+    "DenizensOfTheDeep": max_number_room_checks+88,
+    "TheUselessTrinket": max_number_room_checks+89,
+}
+
+
+location_table_fates_events = {
+    'IsThereNoEscape?Event': None,
+    'DistantRelativesEvent': None,
```

Comment:
This uses single quotes, everywhere else you're using double quotes. Would recommend swapping to doubles here for consistency.